### PR TITLE
Master branch with gcc10 support

### PR DESCRIPTION
**Pull request by issuehsu**

### CODE REVIEW

**Comment by issuehsu** [comment]
Need to be merged after https://github.com/sifive/freedom-e-sdk/pull/498

File: scripts/standalone.mk [hunk 1]
```diff
@@ -219,18 +219,19 @@ ifeq ($(RISCV_SERIES),sifive-8-series)
 RISCV_XCFLAGS += -O2 -fno-common -funroll-loops -finline-functions -funroll-all-loops --param max-inline-insns-auto=20 -falign-functions=8 -falign-jumps=8 -falign-loops=8 --param inline-min-speedup=10 -mtune=sifive-7-series -ffast-math
 endif
 ifeq ($(RISCV_SERIES),sifive-7-series)
-RISCV_XCFLAGS += -O2 -fno-common -funroll-loops -finline-functions -funroll-all-loops --param max-inline-insns-auto=20 -falign-functions=8 -falign-jumps=8 -falign-loops=8 --param inline-min-speedup=10 -mtune=sifive-7-series -ffast-math
-endif
-ifneq ($(RISCV_SERIES),sifive-8-series)
```
Comment (bsousi5) [inline scripts/standalone.mk:224]: Removing this will break 8-series

File: scripts/standalone.mk [hunk 2]
```diff
@@ -219,18 +219,19 @@ ifeq ($(RISCV_SERIES),sifive-8-series)
...
 RISCV_XCFLAGS += -O2 -fno-common -funroll-loops -finline-functions -falign-functions=16 -falign-jumps=4 -falign-loops=4 -finline-limit=1000 -fno-if-conversion2 -fselective-scheduling -fno-tree-dominator-opts -fno-reg-struct-return -fno-rename-registers --param case-values-threshold=8 -fno-crossjumping -freorder-blocks-and-partition -fno-tree-loop-if-convert -fno-tree-sink -fgcse-sm -fno-strict-overflow
 else
 RISCV_XCFLAGS += -O2 -fno-common -funroll-loops -finline-functions -falign-functions=16 -falign-jumps=4 -falign-loops=4 -finline-limit=1000 -fno-if-conversion2 -fselective-scheduling -fno-tree-dominator-opts
 endif
 endif
-endif
 RISCV_XCFLAGS += -DITERATIONS=$(TARGET_CORE_ITERS)
+GCC_VER_GTE10 := $(shell echo `gcc -dumpversion | cut -f1-2 -d.` \>= 10 | bc )
+ifeq ($(GCC_VER_GTE10),1)
+RISCV_XCFLAGS += -fno-tree-loop-distribute-patterns --param fsm-scale-path-stmts=3
```
Comment (bsousi5) [inline scripts/standalone.mk:233]: Is this special for GCC 10? Can you add a comment explaining the purpose

File: scripts/standalone.mk [hunk 3]
```diff
@@ -214,24 +217,22 @@ RISCV_XCFLAGS += -DDHRY_ITERS=$(TARGET_DHRY_ITERS)
 endif
 
 ifeq ($(PROGRAM),coremark)
-ifeq ($(RISCV_SERIES),sifive-8-series)
+ifeq ($(RISCV_SERIES),$(filter $(RISCV_SERIES),sifive-7-series sifive-8-series))
 # 8-series currently uses 7-series mtune, but this may change
```
Comment (bsousi5) [inline scripts/standalone.mk:221]: Can you remove the above comment

**Pull request merged**